src/main/scala/piconot/Empty.scala

```diff
@@ -0,0 +1,24 @@
+package piconot
+
+object Empty extends PicoBot {
+
+  use map "resources/empty.txt"
+
+  val starting, goEast, downSweep, upSweep = States
+
+  starting-
```

Comment:
I agree that `<statename>-` isn't as pretty, but it is definitely a reasonable alternative. Nice!